packages/cli/package.json

```diff
@@ -70,17 +70,17 @@
   "peerDependencies": {
     "@angular/cli": "^13.2.1",
     "@vue/cli": "^4.5.11",
-    "create-react-app": "latest"
+    "create-react-app": "*"
```

Comment:
Why this change ?

Reply:
Mh, yarn did that while I was playing locally, will revert.